core/components/molecules/tabs/tabs.md

```diff
@@ -4,32 +4,17 @@
 ```
 
 ```jsx
-<Tabs>
+<Tabs selected={0} onSelect={() => {}}>
```

Comment:
My only comment would be to add a complete working example with state here. This is to show the Tabs working as we intend them too. Otherwise, it looks like a bug :)

Reply:
We have a fully working example with state below the prop table 😄

Reply:
Yep, my thought is to have a similar working example for the prop table example (without the input).

Reply:
Okay let's do it 👌